test/infrastructure/vcsim/controllers/vcsim_controller.go

```diff
@@ -231,15 +223,6 @@ func (r *VCenterSimulatorReconciler) reconcileNormal(ctx context.Context, vCente
 		if err := vmoperator.ReconcileDependencies(ctx, r.Client, dependenciesConfig); err != nil {
 			return err
 		}
-
-		// The vm-operator doesn't take care of the networking part of the VM, which is usually
```

Comment:
Can you give me a tl;dr why we don't need this anymore? I don't have a good overview to be honest

Reply:
This was just a workaround to trickle down some info to other vcsim controllers.
I have now simplified all of this and our controllers are reading those info from the same config map/secret the vm-operator is using